pgql-tests/error-messages/graph_table.spt

```diff
@@ -109,12 +109,13 @@ test Restriction: LIMIT outsie GRAPH_TABLE [[
 
 test Restriction: projecting vertex/edge objects [[
 
-  SELECT *
+  SELECT [[*]]
```

Comment:
these three error messages moved to a less ideal location because of the transformation, but I think that should still be OK